spec/models/druid_version_zip_spec.rb

```diff
@@ -5,12 +5,38 @@
   let(:druid) { 'bj102hs9687' }
   let(:version) { 1 }
 
+  describe '#base_key' do
+    it 'returns filename without ext' do
+      expect(dvz.base_key).to eq 'bj/102/hs/9687/bj102hs9687.v0001'
+    end
+  end
+
   describe '#s3_key' do
     it 'returns a tree path-based key' do
       expect(dvz.s3_key).to eq 'bj/102/hs/9687/bj102hs9687.v0001.zip'
     end
   end
 
+  describe '#ensure_zip_directory' do
+    it 'returns path if zip directory exists' do
+      expect(dvz.ensure_zip_directory!.to_s).to eq '/tmp/bj/102/hs/9687/bj102hs9687.v0001.zip'
+    end
+
+    context 'zip directory does not exist' do
+      before {
+        FileUtils.rm_rf('/tmp/bj/102/hs/9687/')
+      }
```

Comment:
this can be all on one line 
` before { FileUtils.rm_rf('/tmp/bj/102/hs/9687/') }`